src/mscorlib/src/System/Globalization/CompareInfo.cs

```diff
@@ -459,6 +491,34 @@ internal static unsafe int CompareOrdinalIgnoreCase(string strA, int indexA, int
                     length--;
                 }
 
+                if (CultureData.InvariantMode)
```

Comment:
Yes thanks I meant 0xFFFF :) ... this should get a good comment, though, as I had to think about it.